app/views/helpers/injected/home/PenaltiesAndAppealsTile.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.components._
+@import models.homePage.PenaltiesAndAppealsTileViewModel
+
+@this(
+    home_tile: home_tile,
+    p: p,
+    link: link
+)
+
+@(penaltiesAndAppealsTileViewModel: PenaltiesAndAppealsTileViewModel)(implicit messages: Messages)
+
+@penaltiesAndAppealsLink = @{
+    link(
+        id = Some("sa-penalties-and-appeals-link"),
+        link = "",
+        messageKey = "home.penaltiesAndAppeals.view"
+    )
+}
+
+@if(penaltiesAndAppealsTileViewModel.penaltyPoints.isDefined) {
+    @home_tile("penalties-and-appeals-tile", messages("home.penaltiesAndAppeals.heading")) {
+        @(penaltiesAndAppealsTileViewModel.penaltyPoints match {
+            case Some(2) =>
```

Comment:
not sure why we are doing this match here, this is threshold which we need be check depending on "Submission frequency" , so you can create a field in the viewModel like:

`case class PenaltiesAndAppealsTileViewModel(penaltyPoints: Option[Int], submittingFrequency: String) {
  val finalPoints: Option[Int] = (submittingFrequency, penaltyPoints) match {
    case ("Annual", Some(points)) if points > 2 => penaltyPoints
    case ("Quarterly",  Some(points)) if points > 4 => penaltyPoints
    case _ => None
  }
}`